code/modules/lighting/lighting_system.dm

```diff
@@ -338,10 +338,7 @@
 		if(newalpha >= LIGHTING_DARKEST_VISIBLE_ALPHA)
 			newalpha = 255
 		if(lighting_object.alpha != newalpha)
-			if(instantly)
```

Comment:
oh, well see, now you have more work, because this instantly arg is now no longer needed. so you need to remove it, and then find everything calling redraw_lighting, and if it passes this arg, remove that from the call as well.

=P